app/controllers/admin/watched_words_controller.rb

```diff
@@ -36,11 +38,14 @@ def upload
       begin
         CSV.foreach(file.tempfile, encoding: "bom|utf-8") do |row|
           if row[0].present? && (!has_replacement || row[1].present?)
-            WatchedWord.create_or_update_word(
+            watched_word = WatchedWord.create_or_update_word(
               word: row[0],
               replacement: has_replacement ? row[1] : nil,
               action_key: action_key
             )
+            if watched_word.valid?
+              StaffActionLogger.new(current_user).log_watched_words_creation(watched_word)
```

Comment:
Do we want to log if the watch word was updated instead of created?

Reply:
I believe it's fine to log update as created otherwise we'll need to add another user action for update type and keep track of watched word updates.